packages/modular-scripts/package.json

```diff
@@ -121,6 +122,8 @@
     "ws": "7.4.6"
   },
   "peerDependencies": {
+    "react": "17.0.2",
```

Comment:
We shouldn't tie `modular-scripts` to a specific version of `react` etc. Instead we can replace these with `^16 || ^17`.